_episodes/09-under-the-hood.md

```diff
@@ -66,7 +71,24 @@ All branches and tags in Git are pointers to commits.
 
 ---
 
-## Demonstration: experimenting with branches
+### Git is basically a content-addressed storage system
+
+- CAS: ["mechanism for storing information that can be retrieved based on its content, not its storage location"](https://en.wikipedia.org/wiki/Content-addressable_storage)
+- content address is the content digest (SHA-1 checksum)
+- stored data does not change - so when we modify commits, we always create new commits
```

Comment:
.  Git doesn't delete these right away, which is why it is very, very hard to lose data once you have committed it once.

Reply:
Thanks! I will submit a note on this.